src/App.vue

```diff
@@ -98,13 +98,158 @@
+                                    <el-form-item key="add" label="Name" prop="name">
+                                        <el-input type="text" v-model="newName.name"></el-input>
+                                    </el-form-item>
+                                </el-form>
+
+                                <div v-if="(dataType == 'table' ? true : false)">
+                                    <table class="db_column_table" cellspacing="0">
+                                        <tr>
+                                            <th colspan="3">
+                                                <span class="required">*</span>
+                                                Columns info
+                                            </th>
+                                        </tr>
+                                        <tr v-for="(row, index) in inputColumn" :key="index">
+                                            <el-form :model="row" :rules="ruleRow">
+                                                <el-form-item>
+                                                    <tr class="db_column_table_item">
+                                                        <td>
+                                                            <el-form-item prop="name">
+                                                                <el-input placeholder="Name" v-model="row.name"></el-input>
+                                                            </el-form-item>
+                                                        </td>
+                                                        <td>
+                                                            <el-form-item prop="type">
+                                                                <el-input placeholder="Type" v-model="row.type"></el-input>
+                                                            </el-form-item>
+                                                        </td>
+
+                                                        <td>
+                                                            <el-button type="danger" plain @click="removeRow(index)">
+                                                                <i class="fa fa-times"></i>
+                                                            </el-button>
+                                                        </td>
+                                                    </tr>
+                                                </el-form-item>
+                                            </el-form>
+                                        </tr>
+                                    </table>
+                                    <el-button plain @click="addColumn()">New column</el-button>
+                                </div>
+                            </div>
+
+                            <div v-else>
+                                <el-form
+                                    v-if="(dataType == 'table' ? true : false)"
```

Comment:
Same here... If you are repeating this check on multiple parts, move this to a computed property and use the computed instead